opm/models/common/quantitycallbacks.hh

```diff
@@ -53,7 +54,7 @@ class TemperatureCallback
     using ResultRawType = decltype(std::declval<IQFluidState>().temperature(0));
 
 public:
-    using ResultType = typename std::remove_const<typename std::remove_reference<ResultRawType>::type>::type;
+    using ResultType = typename std::remove_const_t<typename std::remove_reference_t<ResultRawType>>;
```

Comment:
Part of the benefit of `remove_const_t` and `remove_reference_t` is that they're already defined in terms of `typename`, meaning you can omit the `typename` keyword at the call site.  In other words, you can write this as
```C++
using ResultType = std::remove_const_t<std::remove_reference_t<ResultRawType>>;
```
instead.

Reply:
indeed. remedied.